pull-data.mjs

```diff
@@ -210,49 +215,81 @@ export default async function run({ github, context, dryRun = false }) {
       ref: tag_name,
     });
 
-    console.log("Collating JSON...");
-
     // @ts-ignore
-    const z = new AdmZip(Buffer.from(zip));
+    const zBuf = Buffer.from(zip)
+    const globFn = glob(zBuf);
+  
 
+    console.group("Collating base JSON...");
     const data = [];
-    for (const f of globZip(z, "*/data/json/**/*.json")) {
-      // The zipball has a top-level directory that we want to ignore
-      const filename = f.entryName.split("/").slice(1).join("/");
-
-      // Break up the JSON into individual objects so we can inject line numbers
-      const objs = breakJSONIntoSingleObjects(f.getData().toString("utf8"));
-      for (const { obj, start, end } of objs) {
-        obj.__filename = filename + `#L${start}-L${end}`;
-        data.push(obj);
+    for (const f of globFn("data/json/**/*.json")) {
+        const filename = f.name.replaceAll("\\", "/");
+        const objs = breakJSONIntoSingleObjects(f.data())
+        for (const { obj, start, end } of objs) {
+            obj.__filename = filename + `#L${start}-L${end}`;
+            data.push(obj);
+        }
+    }
+    console.log(`Found ${data.length} objects.`);
+    console.groupEnd();
+
+
+    console.group("Collating mods JSON...");
+    /** @type {Record<string, { info: any, data: any[] }>} */
+    const dataMods = {};
+    for (const i of globFn("data/mods/*/modinfo.json")) {
+      const modname = i.name.replaceAll("\\", "/").split("/")[2];
+      if (modname === "dda") continue;
```

Comment:
i'd prefer to ignore the mod directory name completely and only read the mod id; can we change this to be a gate on `modId` instead?

and, come to think of it, i think we should also include the MOD_INFO for the dda mod in the bundle :) we can exclude it at display time.